### ISSUE TITLE
RQ task queue

### ISSUE THREAD

**Issue by danrschlosser** [adicu/eventum#114]
<a href="https://github.com/alanhdu"><img src="https://avatars.githubusercontent.com/u/1914111?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [alanhdu](https://github.com/alanhdu)**
_Friday Nov 14, 2014 at 20:26 GMT_
_Originally opened as https://github.com/danrschlosser/eventum/pull/114_

----

Works in development, need to figure out how to actually deploy Redis and RQ.

----

_**[alanhdu](https://github.com/alanhdu)** included the following code: https://github.com/danrschlosser/eventum/pull/114/commits_

**Comment by danrschlosser** [comment]
<a href="https://github.com/natebrennand"><img src="https://avatars.githubusercontent.com/u/2801090?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [natebrennand](https://github.com/natebrennand)**
_Sunday Nov 23, 2014 at 20:15 GMT_

----

Would be nice to have a README section, or something that is super explicit about how to write a task.
Could also be a good move to have a decorator that  logs the tasks (like their start and end times).

**Comment by danrschlosser** [comment]
<a href="https://github.com/natebrennand"><img src="https://avatars.githubusercontent.com/u/2801090?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [natebrennand](https://github.com/natebrennand)**
_Sunday Nov 23, 2014 at 20:17 GMT_

----

I think we'd want to have a consumer of the [failed queue](http://python-rq.org/docs/exceptions/) to insert them into a mongo collection for easy inspection.

Thinking about this in terms of handling emails, we might want to log those into the database as well so that they can be navigated via a web UI.

**Comment by danrschlosser** [comment]
<a href="https://github.com/natebrennand"><img src="https://avatars.githubusercontent.com/u/2801090?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [natebrennand](https://github.com/natebrennand)**
_Sunday Nov 23, 2014 at 20:17 GMT_

----

thoughts?
cc @danrschlosser

**Comment by danrschlosser** [comment]
<a href="https://github.com/danrschlosser"><img src="https://avatars.githubusercontent.com/u/2433509?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [danrschlosser](https://github.com/danrschlosser)**
_Monday Nov 24, 2014 at 01:22 GMT_

----

Let's put this on :yellow_heart: (hold)

I think we shouldn't merge this until we have it tested with an actual task that works.  We should build on top of this with:

1. A task that merits use of RQ (Google Calendar, Emails, other things)
2. Some interface for managing those tasks on the web
